crates/symbolicator/src/services/symbolication/mod.rs

```diff
@@ -2487,11 +2488,30 @@ impl SymbolicationActor {
     }
 }
 
-fn map_symbolic_registers_breakpad(x: BTreeMap<&'_ str, RegVal>) -> BTreeMap<String, HexValue> {
+/// Normalizes the breakpad register names to canonical names as returned by rust-minidump.
+fn map_breakpad_register_name(name: &str, family: CpuFamily) -> String {
+    let canonical = match name {
+        "r11" if family == CpuFamily::Arm32 => "fp",
+        "r13" if family == CpuFamily::Arm32 => "sp",
+        "r14" if family == CpuFamily::Arm32 => "lr",
+        "r15" if family == CpuFamily::Arm32 => "pc",
+        "eflags" => "efl",
+        "x29" => "fp",
+        "x30" => "lr",
+        name => name,
+    };
```

Comment:
working off of the comment for the function, are these mappings pulled from some specific place in `rust-minidump` somewhere? can you include some sort of reference to where this is being pulled from?

Reply:
Its just conventions which might be different in some cases.

Speaking of conventions, it looks like `eflags` is the more widely used variant though.
Interesting also that rust-minidump does not expose it in the x64 context at all.

See https://en.wikipedia.org/wiki/FLAGS_register

Reply:
symbolic and breakpad also used to use the `eflags` name.
https://github.com/getsentry/symbolic/blob/7294a852dec94e70918ff60b1cf9804dcaa6df96/symbolic-minidump/cpp/data_structures.cpp#L239
https://github.com/getsentry/breakpad/blob/f12b6897897fb5b7e74437371fdf2b36a36f5661/src/processor/dump_context.cc#L490

Reply:
However, the `stackwalk_minidump` CLI uses `efl`:

https://github.com/getsentry/breakpad/blob/09b056975dacd1f0f815ad820b6dc9913b0118a3/src/processor/stackwalk_common.cc#L311